rest_framework_simplejwt/views.py

```diff
@@ -37,7 +38,7 @@ class TokenObtainPairView(TokenViewBase):
     token pair to prove the authentication of those credentials.
     """
 
-    serializer_class = serializers.TokenObtainPairSerializer
+    serializer_class = api_settings.TOKEN_OBTAIN_SERIALIZER
```

Comment:
Does DRF have a method for grabbing the serializer class? I feel like importing all serializers during settings initliaization would be a performance hit.

Reply:
do you have any good way to about this?

Reply:
Yea, GenericAPIView has a get_serializer_class method. Override that method and use import_string to get the serializers.

Reply:
so, should I override that method in each subclass view?

Reply:
no. Change serializer_class to be a different attribute that utilizes the API_SETTINGS, perhaps just with an _ prefix. Don't do the string imports in the settings. Make an abstract view that grabs the attribute and import like that